pages/docs/support/permissions.mdx

```diff
@@ -1,11 +1,13 @@
 ---
 title: "Permissions"
-date: "2021-02-01"
+date: "2021-07-20"
 pullQuote: "Grouparoo has a robust permission system you can use to limit access to your employees and applications."
 ---
 
 Grouparoo has a robust permission system you can use to limit access to your employees and applications. The Grouparoo permission model operates on Teams (for individual users) and API Keys (for applications). Consider making multiple Teams and API Keys for each use case.
 
+The first team you create will have full read and write access to all features and capabilities. After you make your first team, you can control which Teams have read and/or write access to different features and capabilities. Controling permissions at the Team level make it easy for different groups of people in your organization to use Grouparoo. For example, an engineer will want access to reading and writing all types of models, while someone on the revenue ops or marketing ops team will likely only want write access to Destinations and Groups.
```

Comment:
This is a really helpful explanation of teams!  Glad you added it.  Do we want to mention that these permissions are only relevant to the UI?  Anyone could go in and directly edit the json files.

Reply:
Yes! Good call.